include/fastdds/rtps/security/logging/Logging.h

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Canonical ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*!
+ * @file Logging.h
+ */
+#ifndef _FASTDDS_RTPS_SECURITY_LOGGING_LOGGING_H_
+#define _FASTDDS_RTPS_SECURITY_LOGGING_LOGGING_H_
+
+#include "fastdds/rtps/security/logging/LogOptions.h"
+#include "fastdds/rtps/security/exceptions/SecurityException.h"
+
+namespace eprosima {
+namespace fastrtps {
+namespace rtps {
+namespace security {
+
+/**
+ * @brief The LoggerListener class
+ */
+class LoggerListener
+{
+  LoggerListener() = default;
+  ~LoggerListener() = default;
+};
+
+/**
+ * @brief Base class for all security logging plugins.
+ */
+class Logging
+{
+public:
```

Comment:
Check other classes in Fast RTPS, make sure we're following the same format (I think these are indented).